services/data_plane_manager/src/main/java/com/futurewei/alcor/dataplane/client/grpc/DataPlaneClientImplV2.java

```diff
@@ -304,7 +308,9 @@ public GrpcChannelStub(ManagedChannel channel, GoalStateProvisionerGrpc.GoalStat
                         .setType(Common.ResourceType.SUBNET)
                         .setId(key)
                         .build();
-                hostResourceBuilder.addResources(subnetResourceIdType);
+                if (!goalStateBuilder.getHostResourcesMap().containsKey(unicastGoalStateV2.getHostIp()) || !goalStateBuilder.getHostResourcesMap().get(unicastGoalStateV2.getHostIp()).getResourcesList().stream().anyMatch(resourceIdType -> resourceIdType.getId().equals(key))) {
```

Comment:
.get(unicastGoalStateV2.getHostIp()) could be null?

Reply:
It might be. First vm don't have neighbor might have empty resource list.